baseline/client/js/jspsych-spatial-orientation.js

```diff
@@ -0,0 +1,142 @@
+                ctx.fillRect(0, 0, canvas.width, canvas.height);
+                ctx.fillStyle = "rgba(0, 0, 0, 1)";
+                ctx.font = "16px sans-serif";
+                ctx.textAlign = "center";
+                // draw circle
+                ctx.setLineDash([]);
+                ctx.beginPath();
+                ctx.arc(centerX, centerY, options.radius, 0, 2 * Math.PI);
+                ctx.stroke();
+                ctx.fillText(options.centerText, centerX, centerY + 20);
+                // draw arrow to top
+                ctx.beginPath();
+                ctx.moveTo(centerX, centerY);
+                ctx.lineTo(centerX, centerY - options.radius);
+                ctx.stroke();
+                ctx.fillText(options.topText, centerX, centerY - options.radius - 20);
+                // draw pointer
+                ctx.setLineDash([5, 5]);
+                ctx.beginPath();
+                ctx.moveTo(centerX, centerY);
+                ctx.lineTo(
+                    centerX + options.radius * Math.cos(pointerAngle + Math.PI/2),
+                    centerY - options.radius * Math.sin(pointerAngle + Math.PI/2)
+                );
+                ctx.stroke();
+                ctx.fillText(
+                    options.pointerText,
+                    centerX + (options.radius+40) * Math.cos(pointerAngle + Math.PI/2),
+                    centerY - (options.radius+40) * Math.sin(pointerAngle + Math.PI/2)
+                );
+            });
+        };
+        let running = true;
+        canvas.addEventListener("mousemove", e => {
+            if (running) {
+                draw(pointerAngleFromMouseEvent(e));
+            }
+        });
+        canvas.addEventListener("click", e => {
+            if (running) {
+                running = false;
+                // build data
+                const responseAngle = pointerAngleFromMouseEvent(e);
+                const clickData = {
+                    responseAngle: responseAngle,
+                    targetAngle: options.targetAngle,
+                };
+                // draw target pointer
+                ctx.setLineDash([]);
+                ctx.strokeStyle = "rgba(255, 0, 0, 1)";
+                ctx.beginPath();
+                ctx.moveTo(centerX, centerY);
+                ctx.lineTo(
+                    centerX + options.radius * Math.cos(options.targetAngle + Math.PI/2),
+                    centerY - options.radius * Math.sin(options.targetAngle + Math.PI/2)
+                );
+                ctx.stroke();
+                // call onClick
+                options.onClick(clickData);
+            }
+        });
+        draw(0);
+    };
+
+    plugin.trial = (displayElement, trial) => {
+        // build and show display HTML
+        {
+            let html = "";
+            html += `<div id="jspsych-spatial-orientation-wrapper">`;
+            html +=     `<div id="jspsych-spatial-orientation-scene">${trial.scene}</canvas>`;
```

Comment:
I think you want `</div>` here, not `</canvas>`.

Reply:
Ah, that's right.